.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,6 +16,7 @@ items that may help during the review.-->
 #### Pre-Review Checklist
 - [ ] Covered the changes with automated tests
 - [ ] Tested the changes locally
+- [ ] (Elastic Only) Built gems and tested that Enterprise Search works well with new gem versions
```

Comment:
```suggestion
- [ ] (Elastic Only) Built gems (`make build_utility`) and tested that Enterprise Search works well with new gem versions
```

We now build 2 gems:
1. make build_utility
2. make build_service

In 8.6, I hope we will only have one gem building process, and this checkbox can be removed

Reply:
Thinking of that, we do need both, right?

Reply:
Assuming crawler code will be fully migrated to `connectors-ruby` in 8.6, where else do we still need the utility gem?

Reply:
`connectors_service` gem includes executables, which will start the app (i.e. start the loop to poll jobs)